geetools/ee_feature_collection.py

```diff
@@ -35,6 +35,46 @@ def toImage(
         width == "" or params.update(width=width)
         return ee.Image().paint(self._obj, **params)
 
+    def toDictionary(
+        self, keyColumn: str | ee.String = "system:index", selectors: list | ee.List = []
+    ) -> ee.Dictionary:
+        """Convert to Dictionary.
+
+        Parameters:
+            keyColumn: the column to use as keys. Must contain unique values, if not it will fail.
+            selectors: a list of properties to add in the output. If the list is empty all properties will be added.
+
+        Returns:
+            a ee.Dictionary with values of keyColumn as keys and ee.Dictionary as
+            values. The output will look like:
+
+            {
+             '00000000000000000010': {'ADM0_CODE': 74578, 'ADM0_NAME': 'Azores Islands'},
+             '00000000000000000011': {'ADM0_CODE': 7, 'ADM0_NAME': 'Andorra'}
+            }
+        """
+        uniqueIds = self._obj.aggregate_array(keyColumn)
+        selectors = ee.List(selectors)
+        keyColumn = ee.String(keyColumn)
+        finalSelectors = ee.List(
+            ee.Algorithms.If(
+                selectors.size().eq(0), self._obj.first().propertyNames(), selectors  # all columns
```

Comment:
About the default behaviour, do you think we should always bring the system property by default ? 
I already mentioned it in the [google issue tracker](https://issuetracker.google.com/issues/329235200) but there is still no answer on this one.

Reply:
ok let's keep everything for now maybe they'll answer me someday.